src/main/java/com/squareup/phrase/Phrase.java

```diff
@@ -214,6 +234,12 @@ public CharSequence format() {
     return formatted;
   }
 
+  /** Set whether span support is enabled or not. */
+  @SuppressWarnings("UnusedDeclaration") // Public API.
+  public void setDisableSpanSupport(boolean disableSpanSupport) {
```

Comment:
invert: `setSpanSupportEnabled`.  default to true.

javadoc should indicate default.  move javadoc from field to this method.